app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    dt_cache = {}
```

Comment:
dt?
please, choose a more descriptive name

Reply:
In regards to your comment, the following piece of code has been revised.